src/main/java/gov/usds/case_issues/db/model/WriteOnceEntity.java

```diff
@@ -47,6 +49,12 @@
 	@Column(updatable=false)
 	private Date createdAt;
 
+	@ManyToOne(optional=true)
```

Comment:
This should be `false`, and we should have a foreign key relationship in the database (or rather, a _lot_ of such relationships) to prove it. That seems like a Tech Debt at this point, however.